views/lobby.ejs

```diff
@@ -187,11 +187,16 @@
 								<label>Mute spectators <input type="checkbox" class="muteSpecs"></label>
 								<br>
 								<label>Mute join/leave messages <input type="checkbox" class="mutejoinleave"></label>
-							
+								<br>
+								<label>Set all highlight colors to yellow <input type="checkbox" class="setHighlightColorsToYellow"></label>
+
 								<br>
 								<br>
 							
 								<button id="playerHighlightColourButton">Click to change highlight colour</button>
+								<br>
+								<br>
+								<button id="removeHighlight" onclick="unhighlightAllPlayers()">Unhighlight all players</button>
```

Comment:
This button should be below the span of individual player colours...
![image](https://user-images.githubusercontent.com/42633156/45153686-3c929a00-b218-11e8-8dd5-6a4f28edad68.png)